.github/workflows/tests.yml

```
@@ -13,7 +13,7 @@ jobs:
 
     strategy:
       matrix:
-        python-version: [3.5, 3.6, 3.7, 3.8]
+        python-version: [3.5, 3.6, 3.7, 3.8, 3.9.0-rc.2 - 3.9]
```

3.9.0 is not yet available for `actions/setup-python@v2`, see https://github.com/actions/python-versions/blob/main/versions-manifest.json

---

```suggestion
        python-version: [3.5, 3.6, 3.7, 3.8, 3.9]
```

Actually it was added a few hours after your comment with [this commit](https://github.com/actions/python-versions/commit/1430296346bab896c21a97ff3396425c26ad87b1) =]